.gitignore

```diff
@@ -4,6 +4,7 @@
 !.gitignore
 !.gitattributes
 !.coveragerc
+!CONTROLS.rst
```

Comment:
Why was this added?

Reply:
Hey! Sorry for the delays; busy with the day job. I think it's because I edited the controls to reflect that [6] would sort by gilded, but it appeared that the CONTROLS.rst file wasn't showing up as updated in my git client. I added this line to make that explicit.